.github/workflows/kind-e2e.yaml

```diff
@@ -50,6 +50,10 @@ jobs:
       DAPR_REGISTRY: localhost:5000/dapr
       DAPR_TAG: dev
       DAPR_NAMESPACE: dapr-tests
+      # Useful for upgrade/downgrade/compatibility tests
+      # 1.9.6 does not work on newest Windows version for E2E tests
+      # Add this env var to Windows runs too once we have an old version that works with latest win version
+      DAPR_TEST_OLD_IMAGE: "ghcr.io/dapr/daprd:1.9.6"
```

Comment:
Thoughts on calling this something like `DAPR_TEST_N_MINUS_2_IMAGE` and then we can also have `DAPR_TEST_N_MINUS_1_IMAGE`?